userdocs/README.md

```diff
@@ -1,3 +1,15 @@
+# Important - Content Migration to AWS Docs
+
+The eksctl docs are live on the AWS Docs platform
+
+* New site: https://docs.aws.amazon.com/eks/latest/eksctl/what-is-eksctl.html
+* New GitHub source: https://github.com/eksctl-io/eksctl-docs
+
+
+This directory is for the `eksctl.io` site. 
+
+The `eksctl.io` site is still live, but in the future it will redirect to the AWS Docs.
+
 
 # Writing and publishing user docs
```

Comment:
Should we update this section with info about updating the docs for the new site?

Reply:
I clarified this is for the old site, and added a link to the contribute page in the EKS docs